src/ekore/anomalous_dimensions/unpolarized/time_like/as1.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+"""The following are the unpolarized time-like leading order Altarelli-Parisi splitting kernels."""
+
+import numba as nb
+import numpy as np
+from eko import constants
+
+
+
+@nb.njit(cache=True)
+def gamma_qq(N, s1):
+    """
+    Computes the LO quark-quark anomalous dimension
+    Implements Eqn. (B.3) from hep-ph/0604160
+
+    Input parameters
+    ----------------
+    N : Mellin moment (type: complex)
+    s1 : harmonic sum $S_{1}$ (type: complex)
+
+    Returns
+    -------
+    gamma_qq : LO quark-quark anomalous dimension $\gamma_{qq}^{(0)}(N)$ (type: complex)
```

Comment:
Here the name is optional, since it is invisible to the user, but you want to keep the type (same reason of above)
```suggestion
    gamma_qq : complex
        LO quark-quark anomalous dimension $\gamma_{qq}^{(0)}(N)$ (type: complex)
```